app/controllers/sites/query_downloads_controller.rb

```diff
@@ -48,7 +48,7 @@ def top_queries
   def ctr(click_count, query_count)
     return '--' if click_count == 0 || query_count == 0
 
-    sprintf("%.1f%", click_count.to_f * 100 / query_count)
+    sprintf('%.1f%%', click_count.to_f * 100 / query_count)
```

Comment:
This was the only code change necessary for the 2.5 upgrade. See https://bugs.ruby-lang.org/issues/13761. This change is supported by both 2.3 and 2.5:
```ruby
> sprintf("%.1f%", click_count.to_f * 100 / query_count)
"150.0%"
> sprintf("%.1f%%", click_count.to_f * 100 / query_count)
"150.0%"
```